manim/mobject/value_tracker.py

```diff
@@ -30,23 +30,39 @@ def construct(self):
                 pointer = Vector(DOWN)
                 label = MathTex("x").add_updater(lambda m: m.next_to(pointer, UP))
 
-                pointer_value = ValueTracker(0)
+                pointer_tracker = ValueTracker(0)
                 pointer.add_updater(
                     lambda m: m.next_to(
-                                number_line.n2p(pointer_value.get_value()),
+                                number_line.n2p(pointer_tracker.get_value()),
```

Comment:
Can you animate `x` as then the example would be more intuitive.

Reply:
I chose pointer_tracker to indicate it is a valuetracker for the pointer, x would be less intuitive in my opinion.